projects/goharbor/harbor-chart-server/Makefile

```diff
@@ -0,0 +1,43 @@
+BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
```

Comment:
Lets break this project out into another PR and start there and try to get this one all the way thru, code + prow jobs + cdk changes + etc.  There looks like some oddities here but for the most part doesn't look to nonstandard so probably a decent one to start with to get through the whole process.

Reply:
particularly this one since we are handling helm charts differently anyway